pkg/awsconfig/testing/firewallConfig.go

```diff
@@ -0,0 +1,410 @@
+	}
+
+	_, err := c.ec2Client.CreateRoute(Ruleinput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			fmt.Println(err.Error())
+		}
+	}
+
+}
+func (c Client) CreateFirewall(FirewallSubnet ec2.CreateSubnetOutput, Vpc ec2.CreateVpcOutput)(networkfirewall.CreateFirewallOutput){
+	RuleGroupinput := &networkfirewall.CreateRuleGroupInput{
+		Capacity:      aws.Int64(100),
+		RuleGroupName: aws.String("test-firewall"),
+		Type:          aws.String("STATEFUL"),
+		RuleGroup: &networkfirewall.RuleGroup{
+			RulesSource: &networkfirewall.RulesSource{
+				RulesSourceList: &networkfirewall.RulesSourceList{
+					GeneratedRulesType: aws.String("DENYLIST"),
+					TargetTypes:        []*string{aws.String("TLS_SNI")},
+					Targets:            []*string{aws.String(".quay.io"), aws.String(".amazonaws.com"), aws.String("api.openshift.com"), aws.String(".redhat.io")},
+				},
+			},
+		},
+	}
+
+	statefulRuleGroup, err := c.firewallClient.CreateRuleGroup(RuleGroupinput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			fmt.Println(err.Error())
+		}
+	}
+	fmt.Println("Successfully creates a Stateful Rule Group")
+	time.Sleep(wait)
+	
+	FirewallPolicyInput := &networkfirewall.CreateFirewallPolicyInput{
+		Description: aws.String("test"),
+		FirewallPolicyName:  aws.String("testPolicy"),
+		FirewallPolicy: &networkfirewall.FirewallPolicy{
+			StatefulRuleGroupReferences: []*networkfirewall.StatefulRuleGroupReference{&networkfirewall.StatefulRuleGroupReference{
+				ResourceArn: statefulRuleGroup.RuleGroupResponse.RuleGroupArn},
+			},
+			StatelessDefaultActions: []*string{aws.String("aws:forward_to_sfe")},
+			StatelessFragmentDefaultActions: []*string{aws.String("aws:forward_to_sfe")},
+		},
+	}
+	testFirewallPolicy, err := c.firewallClient.CreateFirewallPolicy(FirewallPolicyInput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			fmt.Println(err.Error())
+		}
+	}
+	fmt.Println("Successfully created a Firewall Policy")
+
+		testFirewallInput := &networkfirewall.CreateFirewallInput{
+		FirewallName:      aws.String("testFirewall"),
+		FirewallPolicyArn: testFirewallPolicy.FirewallPolicyResponse.FirewallPolicyArn,
+		SubnetMappings: []*networkfirewall.SubnetMapping{&networkfirewall.SubnetMapping{
+			SubnetId: FirewallSubnet.Subnet.SubnetId},
+		},
+		VpcId: aws.String(*Vpc.Vpc.VpcId),
+	}
+
+	Firewall, err := c.firewallClient.CreateFirewall(testFirewallInput)
+	if err != nil {
+		if aerr, ok := err.(awserr.Error); ok {
+			switch aerr.Code() {
+			default:
+				fmt.Println(aerr.Error())
+			}
+		} else {
+			fmt.Println(err.Error())
+		}
+	}
+	fmt.Println("Successfully created a Firewall. Having to wait a long time for the endpoint to be ready!")
```

Comment:
See above. I'd suggest `Successfully created a Firewall. Waiting 6 minutes for endpoint to become ready...`